SConstruct

```diff
@@ -1032,7 +1034,9 @@ if env["vsproj"]:
 if env["compiledb"]:
     if env.scons_version < (4, 0, 0):
         # Generating the compilation DB (`compile_commands.json`) requires SCons 4.0.0 or later.
-        print_error("The `compiledb=yes` option requires SCons 4.0 or later, but your version is %s." % scons_raw_version)
+        print_error(
+            "The `compiledb=yes` option requires SCons 4.0 or later, but your version is %s." % scons_raw_version
+        )
```

Comment:
Unintended change?

Reply:
Surprisingly, no. This line was too long so Ruff changed it, but it somehow got committed as-is without issue. So while *unrelated*, this would need to be committed eventually.

In fact, it *was* committed in the recent clang-cl PR, so this will harmlessly disappear in the merge process. Though, I might as well rebase this soon to outright remove it